documentation/jetty-documentation/src/main/asciidoc/programming-guide/migration/migration-11-to-12.adoc

```diff
@@ -49,8 +49,52 @@
 [[pg-migration-11-to-12-servlet-to-handler]]
 ==== Migrate Servlets to Jetty Handlers
 
+Web applications written using the Servlet APIs may be re-written using the Jetty `Handler` APIs.
+The sections below outline the Jetty `Handler` APIs that correspond to the Servlet APIs.
```

Comment:
We probably need to say a bit more here about shy you would do this and the pros and cons of doing so... something like:

```suggestion
Prior to Jetty version 12, Jetty only supported the standard Servlet API for requests and responses, which could be handled either in a Jetty Handler or by a standard Servlet.
From Jetty version 12 onwards there is now a core API for requests and responses, together with an updated API for jetty Handlers.     Writing to the Servlet APIs has the benefit of using standard API's, but it does carry some overhead. The core API's are simple and minimalist, giving low overheads, but are intrinsically asynchronous and thus require more rigorous adherence to the API contract.  
Web applications written using the Servlet APIs may be re-written using the Jetty `Handler` APIs.
The sections below outline the Jetty `Handler` APIs that correspond to the Servlet APIs.
```

Reply:
I like Greg's suggested change. It's helpful to know when one should favor using one API vs the other.